test/compile.jl

```diff
@@ -32,8 +39,12 @@ try
 
     # use _require_from_serialized to ensure that the test fails if
     # the module doesn't load from the image:
-    println(STDERR, "\nNOTE: The following 'replacing module' warning indicates normal operation:")
```

Comment:
since the third parameter is `true`, this command is getting broadcast to all of the workers as well (but they have also already loaded it because of the call to `Base.require` earlier), so everyone is printing the same message.